ern-container-gen-android/src/AndroidGenerator.ts

```diff
@@ -707,7 +691,49 @@ You should replace "${annotationProcessorPrefix}:${dependency}" with "annotation
           'lib/src/main/java/com/walmartlabs/ern/container/plugins',
         );
         shell.cp(pathToPluginConfigHook, pathToCopyPluginConfigHookTo);
+
+        this.updatePluginsToAndroidxImports(
+          rnVersion,
+          plugin,
+          pathToCopyPluginConfigHookTo,
+        );
+      }
+    }
+  }
+
+  /**
+   * Modify Plugins to support Androidx
+   * eg: import android.support.annotation.nonNull => import androidx.annotation.nonNull
+   */
+  public updatePluginsToAndroidxImports(
+    rnVersion: string,
+    plugin: PackagePath,
+    outDir: string,
+  ) {
+    if (semver.gte(rnVersion, '0.60.0')) {
```

Comment:
A few suggested changes for this function:

Move the if check _outside_ the function, i.e. no _not_ pass `rnVersion` into the function (that logic should not be part of it).
The only reason `plugin` is passed is to log it - Logging is also something that should _not_ be part of this function - Especially now that it is used in two separate places (the message would be inaccurate and misleading when used on the plugin _hook_ files). Instead, what I suggest is to **return the number of processed files** (`filesWithSupportLib.length`) from this function, and then perform any potential logging on the caller side.
The only remaining parameter to the function should be `outDir` (also suggest so rename it to simply `dir`).